apps/projects/app/components/Card/Issue.js

```diff
@@ -27,8 +27,27 @@ const labelsTags = (labels, theme) =>
     </Tag>
   ))
 
+const FlexibleDiv = ({ compact, children }) => {
+  return compact ? (
+    <div>
+      {children}
+    </div>
+  ) : (
```

Comment:
I find ternaries like this a bit confusing. Why not this?

```js
if (compact) return <div>children</div>

return (
  <React.Fragment>
    {dot}
    {children}
  </React.Fragment>
)
```

I'm also wondering if we could do something like this instead:

```js
return (
  <React.Fragment>
    {compact ? <br /> : dot}
    {children}
  </React.Fragment>
)
```